descope/management/user.py

```diff
@@ -0,0 +1,144 @@
+from typing import List
```

Comment:
I think we are missing here the batch operations capabilities like delete multiple user at one operation or more common use case like adding tenant/s and roles to multiple users (scenario like new tenant created and now i would like to add it to all my admins/support users, etc).
We didnt do that at the beginning of authz and the first question was: how to do this in batch.. :) )

Reply:
This is not a requirement yet. When we'll get to the bridge...

Reply:
but it obviously needed.. I think we must have this (even for the scenario I raised here..),
anyway, we can merge this and add it in another PR.